### PR TITLE
Added PhoenixMiner as a new Ethereum mining option

### PR DESCRIPTION
**Pull request by TimTinkers**
PhoenixMiner is also preferred over Claymore when mining Ethereum on compatible cards.

### CODE REVIEW

File: packages/web-app/src/modules/salad-bowl/PluginDefinitionFactory.ts [hunk 1]
```diff
@@ -27,10 +30,14 @@ export const getPluginDefinitions = (store: RootStore): PluginDefinition[] => {
 
   // Ethereum / Ethash
   if (preferNiceHash && has4gbSupport) {
+    pluginDefinitions.push(getPhoenixMinerEthashDefinition(machine)) // NiceHash
     pluginDefinitions.push(getClaymoreEthashDefinition(machine)) // NiceHash
```
Comment (seniorquico) [inline packages/web-app/src/modules/salad-bowl/PluginDefinitionFactory.ts:34]: Why should we keep Claymore in the list?

File: packages/web-app/src/modules/salad-bowl/definitions/getPhoenixMinerEthashNanopoolDefinition.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Machine } from '../../machine/models/Machine'
+import { PluginDefinition } from '../models'
+import { STANDARD_ERRORS, ETH_WALLET_ADDRESS } from './constants'
+
+export const getPhoenixMinerEthashNanopoolDefinition = (machine: Machine): PluginDefinition => {
+  let def = {
+    name: 'PhoenixMiner-Nanopool-Ethash',
```
Comment (seniorquico) [inline packages/web-app/src/modules/salad-bowl/definitions/getPhoenixMinerEthashNanopoolDefinition.ts:7]: All of these plugins have different names, which will currently lead to multiple downloads/copies installed. This is problematic for users' AV/firewalls. Can we either prioritize that rework, or can we make these (temporarily) use the same name?

File: packages/web-app/src/modules/salad-bowl/PluginDefinitionFactory.ts (hunk 1 above)
Comment (TimTinkers) [reply packages/web-app/src/modules/salad-bowl/PluginDefinitionFactory.ts:34]: It's still works quite well for most of our users and would make a good option to fall back to.

File: packages/web-app/src/modules/salad-bowl/definitions/getPhoenixMinerEthashNanopoolDefinition.ts (hunk 2 above)
Comment (TimTinkers) [reply packages/web-app/src/modules/salad-bowl/definitions/getPhoenixMinerEthashNanopoolDefinition.ts]: All are renamed to PhoenixMiner now.

**Pull request merged**